app/models.py

```diff
@@ -353,6 +353,36 @@ def __repr__(self):
         return '<MatomoDailyGetDatasetPageViewsSummary {}>'.format(self.id)
 
 
+class MatomoDailyGetPortalDownloadSummary(db.Model):
+    """
+    Provides Matomo daily download summary to store the statistics
+    on the different downloads via the portal.
+
+    Chosen variables from the endpoint are:
+    - url              = page URL
+    - label            = page label
+    - nb_hits          = number of views on this page
+    - nb_visits        = number of visits (30 min of inactivity considered a new visit)
+    - nb_uniq_visitors = number of unique visitors
+    - sum_time_spent   = total time spent on this page, in seconds
+    - segment          = segment with download URL
+    """
+    __tablename__ = 'matomo_daily_portal_download_summary'
+
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    date = db.Column(db.String(12))
```

Comment:
Have you considered the db.Date type here? https://docs.sqlalchemy.org/en/13/core/type_basics.html#sqlalchemy.types.Date

Reply:
yes. It was the first thing I did when I created the Matomo tables on another PR but the `flask db upgrade` command failed when the date field was db.Date so I had to change it to string unfortunately. I don't remember the exact details of the failure but it was the only solution to fix the failure...